x-pack/plugins/enterprise_search/public/applications/workplace_search/utils/is_not_nullish.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export function isNotNullish<T>(value: T | null | undefined): value is T {
+  return value !== null && value !== undefined;
```

Comment:
Don't change, just a note. `value != null` works exactly the same and quite common in a lot of places, not sure how TS handles that though.

Reply:
TS handles that fine but I'm not a huge fan of that optimisation because it's not immediately obvious what it does if you don't already know. This is more wordy but also more explicit, which is generally better imo.